src/wallet/wallet.h

```diff
@@ -67,7 +68,8 @@ static const unsigned int DEFAULT_TX_CONFIRM_TARGET = 6;
 //! -walletrbf default
 static const bool DEFAULT_WALLET_RBF = false;
 //! Largest (in bytes) free transaction we're willing to create
-static const unsigned int MAX_FREE_TRANSACTION_CREATE_SIZE = 0;
+//mlumin: 5/2021 increase max free tx create size to 1000 bytes
```

Comment:
to prevent negative CAmount in src/dogecoin-fees.cpp L81? Or for another reason?

Reply:
See wallet.cpp:2668 - this enables free transactions. Going to revert as a policy change to come into the next PR.